simple/stats/cache.py

```diff
@@ -45,48 +53,65 @@ def __init__(self, svg_triples: list[Triple], sv_triples: list[Triple],
 
   def _create_svg_nodes(self, svg_triples: list[Triple],
                         specialized_names: ParentSVG2ChildSpecializedNames):
+    """
+    Creates StatVarGroupNode protos from the SVG triples.
+    Creates ChildSVG instances in these nodes based on specializationOf predicates.
+    If specialized names are available, they are used to set the specialized_entity field in the ChildSVG instances.
+    """
     for triple in svg_triples:
       svg_id = triple.subject_id
       svg_node = self._get_or_create_svg_node(svg_id)
-      value = triple.object_id or triple.object_value
-      if not value:
+      object_id = triple.object_id
+      object_value = triple.object_value
+      if not object_id and not object_value:
         continue
```

Comment:
nit: this continue feels unnecessary now?

Reply:
True. Removed.